src/mf/mf-hid.cpp

```diff
@@ -476,6 +476,8 @@ namespace librealsense
 
                             SysFreeString(fName);
                         }
+                        //Releasing resources
+                        safe_release(pSensor);
```

Comment:
I thought it was the `pSensor.p` being released, isn't it?

Reply:
It works well this way (no leakage), and using the pSensor.p leads to: error C2228	left of '.Release' must have class/struct/union